battle-engine.js

```diff
@@ -510,6 +506,16 @@ BattlePokemon = (function () {
 		}
 		return null;
 	};
+	BattlePokemon.prototype.isSuppressing = function (effectType) {
```

Comment:
I'm referring to in game vs PS since my PR is definitely wrong based on the subtleties mentioned earlier.

I'll get to all this stuff eventually...